src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1129,7 +1150,8 @@ private static byte[] getData(Profile p, int tagSignature) {
      * The array should contain data in a format, corresponded to the
      * {@code tagSignature} as defined in the ICC specification, section 10.
      * This method is useful for advanced applications which need to access
-     * profile data directly.
+     * profile data directly. Only non-BuiltIn profiles can be updated using
+     * this method.
```

Comment:
@prrace 
I added this additional line here mentioning that only non-BuiltIn profiles can be updated. Does it look okay or do you recommend any changes ?

Reply:
I think here also language should be similar to what we use in `@throws IllegalArgumentException`. Using just "non-BuiltIn profiles" doesn't give full picture.

Reply:
@prrace @jayathirthrao  Thanks! Updated both the PR as per CSR changes.